packages/forms/src/UIForm/fieldsets/CollapsibleFieldset/CollapsibleFieldset.scss

```diff
@@ -8,10 +8,18 @@
 	transition: box-shadow 0.3s cubic-bezier(0.25, 0.8, 0.25, 1);
 	margin-bottom: $padding-normal;
 
+	.actions button {
+		opacity: 0;
+	}
+
 	&:hover {
```

Comment:
It's not sufficient to display the buttons on hover for a11y.
In components, we use ally-focus-within that set a classname on all node that contains an element that has the focus. I think we should do the same here.

Reply:
out of scope, we already had the issue before